paddlenlp/transformers/ctrl/modeling.py

```diff
@@ -0,0 +1,764 @@
+# coding=utf-8
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+# Copyright 2018 Salesforce and HuggingFace Inc. team.
+# Copyright (c) 2018, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" paddle2.x CTRL model."""
+
+import numpy as np
+import paddle
+import paddle.nn as nn
+import paddle.nn.functional as F
+from paddle.nn import CrossEntropyLoss, MSELoss
+from .. import PretrainedModel, register_base_model
+
+from .utils import (
+    BaseModelOutputWithPast,
+    CausalLMOutputWithPast,
+    Config,
+    SequenceClassifierOutput, )
```

Comment:
Please don't use such output format. Use PaddleNLP-style output. See other model for example

Reply:
done